Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -226,8 +245,7 @@ def settings_changed(self, *args, **kwargs):
     def _change_name(self, instance, output):
         if instance:
             instance.name = self.effective_learner_name()
-            if self.auto_apply:
-                output.send(instance)
```

Comment:
This was incorrect because it didn't set dirty flag, but was necessary due to broken signal handling.